### ISSUE TITLE
arc-cli missing from (homebrew) release

### ISSUE THREAD

**Issue by oschrenk** [prasmussen/chrome-cli#93]
The README states that 

`brew install chrome-cli` installs the `arc-cli` script.

This is false as the script is not part of the 1.9.3 release

Homebrew points to

https://github.com/Homebrew/homebrew-core/blob/6042d6807d1618529f323af8de46590b780290a3/Formula/c/chrome-cli.rb

the formula does not link the arc-cli script. Which it actually can't since 

it points to 

https://github.com/prasmussen/chrome-cli/archive/refs/tags/1.9.3.tar.gz

which does not contain arc-cli.

Is there a 1.9.4 release planned that includes arc-cli? Once released I can work on a PR in homebrew to get arc-cli included in the formula

**Comment by prasmussen** [comment]
Good point. 1.10.0 has been tagged now. Thanks for helping out.

**Comment by oschrenk** [comment]
Waiting on https://github.com/Homebrew/homebrew-core/pull/175524

**Comment by oschrenk** [comment]
Merged
